apps/web/components/CheckoutPayment/__tests__/CheckoutPayment.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { mount } from '@vue/test-utils';
+import CheckoutPayment from '~/components/CheckoutPayment/CheckoutPayment.vue';
+
+describe('<CheckoutPayment />', () => {
+  it('should render component', () => {
+    const { getByTestId } = mount(CheckoutPayment);
+
+    expect(getByTestId('checkout-payment'));
```

Comment:
Question: I was wondering about this lately and maybe you are able to respond: shouldn't we do any kind of assertion here, like:
```suggestion
    expect(getByTestId('checkout-payment')).to.exist;
```

Reply:
There's no need for such an assertion. There are two methods `getByTestId` and `findByTestId`. Generally those two works the same, difference is that `find` will return empty value and `get` will throw an error and the test will not pass.
There is also a recommendation from [vue test utils](https://test-utils.vuejs.org/api/#get):
```
As a rule of thumb, always use get except when you are asserting something doesn't exist. In that case use [find](https://test-utils.vuejs.org/api/#find).
```

Reply:
yup, thanks for reminding!